src/librustdoc/clean/mod.rs

```diff
@@ -2292,14 +2292,16 @@ impl Clean<Item> for (&hir::MacroDef<'_>, Option<Symbol>) {
             if matchers.len() <= 1 {
                 format!(
                     "{}macro {}{} {{\n    ...\n}}",
-                    vis.print_with_space(cx.tcx, def_id, &cx.cache),
+                    // FIXME(camelid): this might create broken links!
+                    vis.print_with_space(cx.tcx, def_id, &cx.cache, 0),
```

Comment:
I believe this has the same behavior as before, but we should probably test it.

Reply:
I think this will break for macros in submodules (e.g. `pub mod inner { pub macro foo() {} }`).

Reply:
Hmm, so it looks like we don't use links in macro visibilities on nightly anyway:

```rust
#![feature(decl_macro)]

pub mod inner1 {
    pub mod inner2 {
        pub mod inner3 {
            pub(in crate::inner1) macro foo() {}
        }
    }
}
```

![image](https://user-images.githubusercontent.com/37223377/110180131-1553be00-7dbe-11eb-8191-229c7a6b3059.png)

So I'm not sure how to test the changes :/

Reply:
> Hmm, so it looks like we don't use links in macro visibilities on nightly anyway:

That sounds like a bug, could you open an issue?

Reply:
Issue: #83000. Nice issue number :)